systemd/ceph-mon@.service

```diff
@@ -21,7 +21,6 @@ PrivateDevices=yes
 ProtectHome=true
 ProtectSystem=full
 PrivateTmp=true
-TasksMax=infinity
```

Comment:
`TasksMax` was introduced by 05cafcf19fb80dd2a76fdbbe5329bae53aa0def4 [0]. seems this setting does not exist in systemd version earlier than 226. and RHEL/Centos7.2 still have v219[1]. so does some relatively old distro releases. debian jessie has systemd v215[2].

---
[0] https://github.com/ceph/ceph/pull/8450#issuecomment-219486514
[1] https://bugzilla.redhat.com/show_bug.cgi?id=1199644
[2] https://packages.debian.org/jessie/systemd